docs/_admin/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Project Administration
+order: -1
+---
+
+The web-platform-tests project is more than just tests! This section documents
```

Comment:
Not sure this first sentencce is adding much.

Reply:
Just a little levity. I'll remove it